lsp-mode.el

```diff
@@ -1877,6 +1897,101 @@ WORKSPACE is the workspace that contains the progress token."
                        (cons status global-mode-string))
                       (t (remove status global-mode-string))))))
 
+
+;; code actions modeline
+
+(defvar-local lsp--modeline-code-actions-string nil
+  "Holds the current code action string on modeline.")
+
+(declare-function all-the-icons-octicon "ext:all-the-icons")
+
+(defun lsp--modeline-code-actions-icon ()
+  "Build the icon for modeline code actions."
+  (if (featurep 'all-the-icons)
+      (all-the-icons-octicon "light-bulb"
+                             :face 'lsp-modeline-code-actions-face
+                             :v-adjust -0.0575)
+    (propertize "💡" 'face 'lsp-modeline-code-actions-face)))
+
+(defun lsp--modeline-build-code-actions-string (actions)
+  "Build the string to be presented on modeline for code ACTIONS."
+  (-let* ((icon (lsp--modeline-code-actions-icon))
+          (first-action-string (propertize (->> actions
+                                                lsp-seq-first
+                                                (gethash "title")
```

Comment:
please use either `lsp:*` or destructuring. Lets write the new code against `lsp-protocol.el`.

Reply:
Should I point this branch to plists branch so?

Reply:
Oh, `lsp:code-actions-title` is on master, done!